library/debug.c

```diff
@@ -410,4 +410,43 @@ void mbedtls_debug_printf_ecdh( const mbedtls_ssl_context *ssl, int level,
 }
 #endif /* MBEDTLS_ECDH_C */
 
+#if defined(MBEDTLS_SSL_PROTO_TLS1_3) && defined(MBEDTLS_SSL_SESSION_TICKETS)
+void mbedtls_debug_print_ticket_flags( const mbedtls_ssl_context *ssl, int level,
```

Comment:
```suggestion
void mbedtls_debug_print_ticket_flags(
     const mbedtls_ssl_context *ssl, int level,
```
Vertical alligned and don't exceed 80 characters in one line.